HW4/task1.py

```diff
@@ -0,0 +1,32 @@
+# Создайте несколько переменных заканчивающихся и не оканчивающихся на «s».
+# Напишите функцию, которая при запуске заменяет содержимое переменных оканчивающихся на s
+# (кроме переменной из одной буквы s) на None.
+# Значения не удаляются, а помещаются в одноимённые переменные без s на конце.
+
+def remove_s():
+    glob_dict = globals()
+    for var in list(glob_dict):
+        if len(var) > 1 and var[-1] == 's' and str(type(glob_dict[var])) != "<class 'function'>":
```

Comment:
Последние условия вам никак не помогает, но мне понятно, зачем вы так сделали. Просто переименуйте функцию и все.
```suggestion
        if len(var) > 1 and var[-1] == 's':
```

Reply:
> Последние условия вам никак не помогает, но мне понятно, зачем вы так сделали. Просто переименуйте функцию и все.

Вроде бы когда я ставлю print(globals()) до и после запуска функции, то видно, что функция остается собой, несмотря на то, что заканчивается на "s" (хотя соседняя переменная 'datas' теперь содержит None): 

![Screenshot 2023-06-25 at 12 25 49](https://github.com/Rusheslav/Python_Deep_HW/assets/108346979/60ca13a4-eb1b-425a-9c5c-5cdbf85ff859)

Значит, условие помогает. Я понимаю, что проще всего переименовать саму функцию на что-то без окончания "s", но, по-моему, это менее универсальный вариант. А что если у меня в коде было до этого еще несколько функций и какие-то из них заканчивались на "s"? Они все превратятся в None.

Reply:
Согласен. Объяснение принято